smoke-tests/smoke-tests.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
 }
 
 test {
-  dependsOn ':javaagent:shadowJar'
+  inputs.files(tasks.findByPath(':javaagent:shadowJar').outputs.files.singleFile)
```

Comment:
Can/should it be `tasks.findByPath(':javaagent:shadowJar').outputs.files`?

Reply:
Seems that it should.